apps/nextjs/src/components/AppComponents/Chat/aila-start/search-params-utils.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Utility functions for handling search parameters and creating prompts
+ */
+
+/**
+ * Creates a starting prompt for lesson planning based on search parameters
+ *
+ * @param keyStage - The educational key stage (e.g., "Key Stage 3")
+ * @param subject - The subject area (e.g., "History")
+ * @param unitTitle - The specific unit title
```

Comment:
Is there an idea of what Aila is meant to do with the unit? Currently there's no explicit instruction in the prompt around units, I don't think.